sigs.yaml

```diff
@@ -174,6 +174,35 @@ sigs:
         teams:
         - name: kubevirt-community
           description: General Discussion
+workinggroups:
+    - dir: wg-arch-s390x
+      name: S390x architecture working group
+      mission_statement: |
+        WG ARCH s390x takes care of all things related to supporting s390x architecture on KubeVirt clusters.
+        This includes all build processes, also support of failing builds or test lanes.
+      label: wg-arch-s390x
+      leadership:
+        chairs:
+        - github: jschintag
+          name: "Jan Schintag"
+          company: IBM
+        - github: cfilleke
+          name: "Cheryl Fillekes"
+          company: IBM
+        - github: vamsikrishna-siddu
+          name: "vamsikrishna-siddu"
+          company: IBM
+    - dir: wg-arch-arm
+      name: ARM architecture working group
+      mission_statement: |
+        WG ARCH ARM takes care of all things related to supporting ARM architecture on KubeVirt clusters.
+        This includes all build processes, also support of failing builds or test lanes.
+      label: wg-arch-arm
```

Comment:
@zhlhahaha hey, here's the entry for the wg arch arm, please chime in here with the things you want to have added, I'll happily integrate them

Reply:
I'd like to mention @awels and @andymcc here since Alex and Andy have a good grasp of multi-arch challenges in K8s & OCP operator implementation.

Reply:
Thanks, Daniel!

Reply:
@cfilleke so you are suggesting that they might take chair or lead roles in the WG?

@awels @andymcc WDYT? Are you interested taking this on?

Reply:
Do we have a definition of what chairs/lead responsibilities are? I want to know that before agreeing.

Reply:
> SIGs must have at least one and ideally two SIG chairs at any given time. SIG chairs are intended to be organizers and facilitators, responsible for the operation of the SIG and for communication and coordination with the other SIGs, the Steering Committee, and the broader community.

[from the Kubernetes SIGs doc](https://github.com/kubernetes/community/blob/master/governance.md#sigs)

if we map that to WG chairs then I figure WG chairs are responsible of WG operation and for communication and coordination with stakeholder SIGs.

@aburdenthehand @fabiand @lyarwood WDYT?

Reply:
@awels we recently updated the membership policy by adding some roles here: https://github.com/kubevirt/community/blob/main/membership_policy.md#special-interest-group-sig-chair

Reply:
If @andymcc agrees I would be okay with being co-chair with him.